app/javascript/components/patient/Patient.js

```diff
@@ -72,6 +76,77 @@ class Patient extends React.Component {
     }
   }
 
+  renderContactInfo() {
+    let contactInfoPatient = this.state.displayHohContactInfo ? this.props.hoh : this.props.details;
+    return (
+      <Col id="contact-information" lg={10} className="col-xxl-12">
+        <div className="section-header">
+          <h4 className="section-title">Contact Information</h4>
+          {this.state.displayHohContactInfo ? '' : this.renderEditLink('Contact Information', 2)}
+        </div>
+        <div className="item-group">
+          {this.state.isMinor && (
+            <div>
+              <span style={{ color: 'red' }}>Monitoree is a minor.</span>
+            </div>
+          )}
+          {this.state.isMinor && this.props.hoh && (
```

Comment:
per slack discussion this should be a link to the HoH page not a toggle.  This function can be removed and put back in the render and this can be changed to a href that is shown conditionally

Reply:
One thing you'll have to check is if the minor is HoH, if that's the case, don't show this link